#!/usr/bin/env python
PACKAGE = "vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#Params pegs
gen.add("loH", int_t, 0, "Low H bound for buoy", 113, 0, 255)
gen.add("loS", int_t, 0, "Low S bound for buoy", 0, 0, 255)
gen.add("loV", int_t, 0, "Low V bound for buoy", 0, 0, 255)
gen.add("hiH", int_t, 0, "High H bound for buoy", 136, 0, 255)
gen.add("hiS", int_t, 0, "High S bound for buoy", 255, 0, 255)
gen.add("hiV", int_t, 0, "High V bound for buoy", 255, 0, 255)

gen.add("Hough1", int_t, 0, "Hough circle param 1", 350, 0, 400)
gen.add("Hough2", int_t, 0, "Hough circle param 2", 13, 0, 400)

gen.add("minContourArea", int_t, 0, "Min Contour Area", 100, 0, 3000)

exit(gen.generate(PACKAGE, "pegs", "pegs"))